akka-docs/src/main/paradox/typed/fsm.md

```diff
@@ -0,0 +1,54 @@
+# Finite state machines
```

Comment:
Hmmm I wonder how to call the section here...
Think it's fine like that? WDYT about "Behaviors are Finite State Machines"?

Reply:
Or Behaviors as Finite state machines?

Reply:
sounds great :) Plz apply